packages/unplugin/src/plugin/core.ts

```diff
@@ -210,6 +214,7 @@ export const unpluginFactory: UnpluginFactory<PandaPluginOptions | undefined> =
       return result
     },
     vite: {
+      name: 'unplugin-panda',
```

Comment:
Typescript was complaining a `name` was needed in order to add the `generateBundle` hook.